### DISCUSSION TITLE
can't restart robot

### DISCUSSION THREAD

**Discussion by taara67** [iRobotEducation/create3_docs#452] [Questions] [answered]
### How are you connecting to your Create 3?

USB Ethernet (ROS 2)

### Computer(s) Model(s) and Operating System(s)

jetson Orin nano   ubuntu20.04

### Which version of ROS 2 is installed on your computer?

Galactic

### Which firmware version is installed on your robot?

G 5.3

### Which RMW is your robot running?

CycloneDDS

### Is the robot connected to a network? If so, what is the network type?

home network

### Are there multiple Create 3 robots connected to your network?

None

### Is multicast enabled?

None

### What is the Adapter Board's USB/BLE Toggle currently switched to?

USB

### Describe your question.

Hello
My goal is to use Jetson Orin Nano to operate Create3 via USB.
Actually, I have achieved operation using ros2 with jetson nano, but it is not working with Orin.
I also asked about this in "ros2 topic not showing up #368", which provided some solutions. Implementing them also presented fundamentally different problems. That's the problem with not being able to restart the robot.

I did some basic setup and tried restarting the robot to see if it can receive topics. Normally, the restart completes with a happy sound, but for some reason, no matter how many minutes I wait, the restart does not complete. Also, red hazards are always visible. At this time, ping 192.168.186.2 was successful.
The power cannot be turned off even by pressing and holding the power button. Rebooting  robot was  retried. At this time, "ping" was failing. It took 30 minutes to restart so I gave up.
What may be the cause? I would like to know the solution

My most recent work is editing the robot's ntpconf and overriding RMW

If you have any questions, please let me know what to check.

**Comment by brianabouchard** [comment, marked as answer]
@taara67 I strongly suspect the cause of your issue is implementing an override RMW profile on your Create 3 robot that contains an interface not available on the Create 3 as discussed on your issue thread.

I would try the following:

Allow battery to drain completely then place the robot on the dock to boot up. Navigate to the web server at 192.168.186.2, delete the contents of the override RMW profile and then restart the application. Based on what you described, I suspect there will be a short time during initial startup where the web server will be accessible and you can do this.

**Comment by taara67** [comment]
Thank you very much for the reply

The interface remained up during the reboot, so I was able to access the webserver and Woverride the RMWprofile.
The reboot was successful.
